CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ execute_process (COMMAND git  show -s --format=%ci HEAD
 )
 
 #Required libraries
-find_package (OpenCV 2.4 REQUIRED)
+find_package (OpenCV)
```

Comment:
Why was this change required, and why remove the `REQUIRED` flag?

Reply:
If I keep `find_package (OpenCV 2.4 REQUIRED)` on Raspbian Buster, cmake fails with

```
CMake Error at CMakeLists.txt:19 (find_package):
  Could not find a configuration file for package "OpenCV" that is compatible
  with requested version "2.4".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/opencv4/OpenCVConfig.cmake, version: 4.1.0
    /usr/share/OpenCV/OpenCVConfig.cmake, version: 3.2.0

```
Indeed, OpenCV 3.2.0 is the default on Raspbian Buster, I have also installed OpenCV 4.1.0 for tests but there is no OpenCV 2.4.

If I try just without `REQUIRED`, the error becomes a warning and cmake succeeds but make fails, probably because cmake did not add anything related to OpenCV in the generated Makefile. So I removed also `2.4` and everything works. There is probably a way to ask cmake for at least version 2.4 but I did not spend more time on this since now most of the users are likely to have at least that version and possibly older versions would work without many changes...

Reply:
Sorry, I just realize that it should be possible to keep the `REQUIRED `flag without specifying the version, I will try that...